diff --git a/vendor/amasty/product-tabs/Api/TabsRepositoryInterface.php b/vendor/amasty/product-tabs/Api/TabsRepositoryInterface.php
index 600182e..e9995ff 100644
--- a/vendor/amasty/product-tabs/Api/TabsRepositoryInterface.php
+++ b/vendor/amasty/product-tabs/Api/TabsRepositoryInterface.php
@@ -2,11 +2,6 @@
 
 namespace Amasty\CustomTabs\Api;
 
-use Amasty\CustomTabs\Api\Data\TabsInterface;
-use Magento\Framework\Api\SearchCriteriaInterface;
-use Magento\Framework\Api\SearchResultsInterface;
-use Magento\Framework\Exception\CouldNotDeleteException;
-use Magento\Framework\Exception\NoSuchEntityException;
 
 /**
  * @api
@@ -14,52 +9,52 @@ use Magento\Framework\Exception\NoSuchEntityException;
 interface TabsRepositoryInterface
 {
     /**
-     * @param TabsInterface $tabs
-     * @return TabsInterface
+     * @param \Amasty\CustomTabs\Api\Data\TabsInterface  $tabs
+     * @return \Amasty\CustomTabs\Api\Data\TabsInterface
      */
-    public function save(TabsInterface $tabs);
+    public function save(\Amasty\CustomTabs\Api\Data\TabsInterface $tabs);
 
     /**
      * @param int $tabId
-     * @return TabsInterface
-     * @throws NoSuchEntityException
+     * @return \Amasty\CustomTabs\Api\Data\TabsInterface
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
      */
-    public function getById(int $tabId): TabsInterface;
+    public function getById(int $tabId): \Amasty\CustomTabs\Api\Data\TabsInterface ;
 
     /**
      * @param string $tabName
-     * @return TabsInterface
-     * @throws NoSuchEntityException
+     * @return \Amasty\CustomTabs\Api\Data\TabsInterface
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
      */
-    public function getByName(string $tabName): TabsInterface;
+    public function getByName(string $tabName): \Amasty\CustomTabs\Api\Data\TabsInterface ;
 
     /**
-     * @param TabsInterface $tabs
+     * @param \Amasty\CustomTabs\Api\Data\TabsInterface  $tabs
      * @return bool
-     * @throws CouldNotDeleteException
+     * @throws \Magento\Framework\Exception\CouldNotDeleteException
      */
-    public function delete(TabsInterface $tabs): bool;
+    public function delete(\Amasty\CustomTabs\Api\Data\TabsInterface $tabs): bool;
 
     /**
      * @param int $tabId
      * @return bool
-     * @throws CouldNotDeleteException
+     * @throws \Magento\Framework\Exception\CouldNotDeleteException
      */
     public function deleteById(int $tabId): bool;
 
     /**
-     * @param SearchCriteriaInterface $searchCriteria
-     * @return SearchResultsInterface
-     * @throws NoSuchEntityException
+     * @param \Magento\Framework\Api\SearchCriteriaInterface $searchCriteria
+     * @return \Magento\Framework\Api\SearchResultsInterface
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
      */
-    public function getList(SearchCriteriaInterface $searchCriteria): SearchResultsInterface;
+    public function getList(\Magento\Framework\Api\SearchCriteriaInterface $searchCriteria): \Magento\Framework\Api\SearchResultsInterface;
 
     /**
-     * @param TabsInterface $tab
+     * @param \Amasty\CustomTabs\Api\Data\TabsInterface  $tab
      *
-     * @return TabsInterface
+     * @return \Amasty\CustomTabs\Api\Data\TabsInterface
      */
-    public function duplicate(TabsInterface $tab): TabsInterface;
+    public function duplicate(\Amasty\CustomTabs\Api\Data\TabsInterface $tab): \Amasty\CustomTabs\Api\Data\TabsInterface ;
 
     /**
      * @param int $storeId
